src/apps/Altinn.AccessManagement/src/Altinn.AccessMgmt.Core/Services/PartyService.cs

```diff
@@ -1,20 +1,22 @@
 using Altinn.AccessManagement.Core.Errors;
-using Altinn.AccessMgmt.Core.Models;
 using Altinn.AccessMgmt.Core.Services.Contracts;
 using Altinn.AccessMgmt.PersistenceEF.Contexts;
 using Altinn.AccessMgmt.PersistenceEF.Extensions;
 using Altinn.AccessMgmt.PersistenceEF.Models;
+using Altinn.AccessMgmt.PersistenceEF.Utils;
 using Altinn.Authorization.Api.Contracts.Party;
 using Altinn.Authorization.ProblemDetails;
 using Microsoft.EntityFrameworkCore;
 
 namespace Altinn.AccessMgmt.Core.Services;
 
 /// <inheritdoc />
-public class PartyService(AppDbContext dbContext, AuditValues auditValues) : IPartyService
+public class PartyService(AppDbContext dbContext) : IPartyService
 {
+    public AuditValues AuditValues { get; set; } = new AuditValues(AuditDefaults.InternalApi, AuditDefaults.InternalApi, Guid.NewGuid().ToString());
```

Comment:
Med audit middleware som Andreas har laget så får man også satt operation id som: 
`Activity.Current?.TraceId.ToString() ?? context.TraceIdentifier`
i stedet for bare en new guid man ikke vil ha noe forhold til i annen app insights logging f.eks. 

Vet ikke om du har tilgang til httpcontext for å få satt tilsvarende her?